addons/web/static/src/core/commands/command_palette.js

```diff
@@ -245,9 +249,14 @@ export class CommandPalette extends Component {
         scrollTo(command, { scrollable: this.listboxRef.el });
     }
 
-    onCommandClicked(index) {
+    onCommandClicked(event, index) {
+        event.preventDefault();
         this.selectCommand(index);
-        this.executeSelectedCommand();
+        const ctrlKey = isMacOS() ? event.metaKey : event.ctrlKey;
+        if (ctrlKey && !this.state.selectedCommand.href) {
+            return;
+        }
```

Comment:
Isn't that handled in "executeSelectedCommand" anyway? As is checks that there is a href (and otherwise simply triggers "executeCommand").

Would remove this if not necessary to avoid extra code-shortcuts.

Reply:
Not exactly, it's there to not execute the action if you 'ctrl+click' on a command without a link inside it.
But I moved it in "exectueCommand", so that 'ctrl+enter' does not execute the action either, and it makes it easier to understand.